cobalt/build/build_info.py

```diff
@@ -143,5 +143,33 @@ def main(output_path, cwd=_FILE_DIR):
 ''')
 
 
+BUILD_ID_HEADER_TEMPLATE = """
+#ifndef _COBALT_BUILD_ID_H_
+#define _COBALT_BUILD_ID_H_
+
+#ifndef COBALT_BUILD_VERSION_NUMBER
+#define COBALT_BUILD_VERSION_NUMBER "{cobalt_build_version_number}"
+#endif  // COBALT_BUILD_VERSION_NUMBER
+
+
+#endif  // _COBALT_BUILD_ID_H_
+"""
+
+
+def build_id_header(output_path):
+  """Writes a cobalt_build_id header file.
+
+  Args:
+    output_path: Location to write Cobalt build ID header to.
+  Returns:
+    0 on success.
+  """
+  with open(output_path, 'w', encoding='utf-8') as f:
+    f.write(
+        BUILD_ID_HEADER_TEMPLATE.format(
+            cobalt_build_version_number=get_build_id()))
+
+
 if __name__ == '__main__':
-  main(sys.argv[1])
+  build_id_header(sys.argv[1])
```

Comment:
nit: I'd prefer both these functions names to be prefixed with `write_`

Even better would be to have the formatting of the string and the writing be in separate functions as that's more easily testable and the functions would be doing only one thing

Reply:
+1 to including `write` in the function names if they're going to write.

Reply:
Added `write_` but i'm not going to do further refactoring here right now, just basic cleanup.